### ISSUE TITLE
basis of record in the IPT: "occurrence" vs "taxon" and "event"

### ISSUE THREAD

**Issue by ManonGros** [gbif/ipt#1549]
The current accepted terms for [basis of record according to TDWG](https://dwc.tdwg.org/terms/#dwc:basisOfRecord) are:  `PreservedSpecimen`, `FossilSpecimen`, `LivingSpecimen`, `MaterialSample`, `Event`, `HumanObservation`, `MachineObservation`, `Taxon`, `Occurrence`.
However, on GBIF.org, not all of them are accepted/interpreted: `Taxon`, `Occurrence` and `Event` are flagged and interpreted as `unknown`.

For some reason, the IPT considers `Occurrence` a valid term but not `Taxon` and `Event`: https://rs.gbif.org/vocabulary/dwc/basis_of_record.xml

This means that several publishers publish data with an `Occurrence` basis of record that are interpreted as `unknown` (see examples [here](https://www.gbif.org/occurrence/search?basis_of_record=UNKNOWN&occurrence_status=present)).

I don't think it makes much sense. Since we already excluded `Taxon` and `Event` from the list of accepted terms in the IPT, I suggest to remove `Occurrence` as well (or have everything).
Perhaps for the next IPT version?

**Comment by mike-podolskiy90** [comment]
@ManonGros https://rs.gbif.org/vocabulary/dwc/basis_of_record.xml says:
A resource describing an instance of the Occurrence class. NOTE: this value is ambiguous and hence should only be used when the resource type is unknown.

**Comment by ManonGros** [comment]
Fair enough, perhaps we should just have "unknown" instead of "occurrence"? it would be more explicit

**Comment by mike-podolskiy90** [comment]
From Tim:
Just bear in mind that https://rs.gbif.org/vocabulary/dwc/basis_of_record.xml is not really ours to change... it's defined in the DwC (https://dwc.tdwg.org/list/#dwc_basisOfRecord) and we just reformat that into the XML flavour for the DwC-A 

I think we should leave it as it is
